addons/l10n_uy_website_sale/tests/test_l10n_uy_website_sale.py

```diff
@@ -0,0 +1,28 @@
+from odoo.tests.common import HttpCase, tagged
+
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+
+
+@tagged('post_install_l10n', 'post_install', '-at_install')
+class TestUi(HttpCase, AccountTestInvoicingCommon):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+    def test_checkout_address_uy(self):
+        self.env.company.country_id = self.env.ref('base.uy').id
+        self.env.ref('base.user_admin').write({
+            'company_id': self.env.company.id,
+            'company_ids': [(4, self.env.company.id)],
+        })
+        self.env['product.product'].create({
+            'name': 'Test Product',
+            'sale_ok': True,
+            'website_published': True,
+        })
+        self.env['ir.config_parameter'].set_param('sale.automatic_invoice', True)
+        self.env['website'].get_current_website().company_id = self.env.company.id
+        user_admin = self.env.ref('base.user_admin')
+        user_admin.company_ids = user_admin.company_ids + self.env.company
+        self.start_tour('/shop', 'shop_checkout_address_uy', login='admin')
```

Comment:
wtf is that? There is probably a setup in website allowing to create easily a brand new website for your test company

Reply:
Just a bad copy paste from other E-com related localisation module :( and since it not going the master as it already added in saas-18.2 [in PR](https://github.com/odoo/odoo/pull/198327)